src/quests.js

```diff
@@ -129,17 +126,22 @@ function getQuestFromTransactionDescription(description) {
  * checkQuestForAcceptance
  *
  * @param {object} transaction
- * @return {bool}
+ * @return {boolean}
  */
 function checkQuestForAcceptance(transaction) {
   Memory.quests = Memory.quests || {};
+  transaction.description = transaction.description || JSON.stringify({
```

Comment:
I don't understand this change.
This sets the description of the transaction to some 'random' value.